dom.bs

```diff
@@ -1568,8 +1568,8 @@ run these steps:
    <a for="event listener">capture</a> is true, then <a for=iteration>continue</a>.
 
    <li><p>If <var>listener</var>'s <a for="event listener">once</a> is true, then
-   <a for=list>remove</a> <var>listener</var> from <var>event</var>'s {{Event/currentTarget}}
-   attribute value's <a for=EventTarget>event listener list</a>.
+   <a>remove an event listener</a> with the <var>event</var>'s {{Event/currentTarget}}
+   and <var>listener</var></a>.
    <!-- Do this before invocation to avoid reentrancy issues. No need to set removed to true since
         each listener in listeners is run once anyway. -->
```

Comment:
I'd reword the second sentence here to say something like "Note that there is no need to set removed to true since listener is run once, but it is done here to share an abstraction."